docs/data-ai/ai/alert.md

```diff
@@ -4,6 +4,121 @@ title: "Alert on inferences"
 weight: 60
 layout: "docs"
 type: "docs"
-no_list: true
-description: "TODO"
+description: "Use triggers to send email notifications when inferences are made."
 ---
+
+At this point, you should have already set up and tested [computer vision functionality](/data-ai/ai/run-inference/).
+On this page, you'll learn how to use triggers to send alerts in the form of email notifications when certain detections or classifications are made.
+
+You will build a system that can monitor camera feeds and detect situations that require review.
+In other words, this system performs anomaly detection.
+Whenever the system detects an anomaly, it will send an email notification.
+
+First, you'll set up data capture and sync to record images with the anomaly and upload them to the cloud.
+Next, you'll configure a trigger to send email notifications when the anomaly is detected.
+
+## Configured a filtered camera
+
+Your physical camera is working and your vision service is set up.
```

Comment:
Add a prerequisites section please, that way this page follows the general pattern we use on other pages